src/main/resources/static/assets/javascripts/cookie-manager.js

```diff
@@ -0,0 +1,179 @@
+$(document).ready(function(){
```

Comment:
Did we get this file from somewhere? Might be nice to have some comments saying what it's based on.

Reply:
I added a comment with a link to the original 👍